src/pivotal-ui/components/meta/dropdown-driven-tabs.js

```diff
@@ -0,0 +1,14 @@
+// automatically attach a click listener to any dropdown-driven tab anchors on the page
+$(function () {
+  $('.dropdown-driven-tabs a').on('click', function (e) {
+    e.preventDefault()
+    var $a = $(this)
+    var $active = $a.closest('.dropdown-driven-tabs').find('.active')
+    if ($active) {
```

Comment:
You're right, thanks for the tip!

Your code sample doesn't _quite_ work because we need to remove the `active` class from the `<li>` that wraps the `<a>`, not the `<a>` itself, but I will clean up the code based on your recommendations.

I thought we might also want to short-circuit the `$a.tab('show')` if the user selects the already active option, but this looks to be unnecessary.

Thanks!